test/e2e/data/infrastructure-docker/v1.3/bases/mp.yaml

```diff
@@ -34,6 +34,8 @@ metadata:
   name: "${CLUSTER_NAME}-dmp-0"
 spec:
   template:
+      # NOTE: If the Kubernetes version is changed in `clusterctl_upgrade_test.go` the image and SHA must be updated here.
+      customImage: "kindest/node:v1.26.4@sha256:f4c0d87be03d6bea69f5e5dc0adb678bb498a190ee5c38422bf751541cebe92e"
```

Comment:
In a bunch of those upgrade tests we have machinepool yamls, but I think we're not actually using them to generate cluster templates in most cases. For 1.0 we don't have those yamls at all.

Reply:
Would it be better to re-include them or just get rid of them at all. As far as I can see they're only currently used in v0.4.0.